tracer/test/Datadog.Trace.Security.Unit.Tests/SecurityCoordinatorTests.cs

```diff
@@ -35,6 +35,31 @@ public void DefaultBehavior()
             secCoord.Should().BeNull();
         }
 
+#if !NETFRAMEWORK
+        [Fact]
+        public void GivenSecurityCoordinatorInstance_WhenScanNullQuery_NoException()
+        {
+            var contextMoq = new Mock<HttpContext>();
```

Comment:
tiny nit: is it possible to reproduce this with `DefaultHttpContext`? If so, it might simplify this (I find Mocks can be very confusing 😄) No worries if not, and def don't spend much time on it either way!

Reply:
Yes, it is possible to reproduce it. The problem is that it would require to add a new assembly reference to the test project, so I decided to mock it.